administration.html

```diff
@@ -0,0 +1,272 @@
+                            <div style="margin-top: 1em">
+                                <button class="button" type="submit">Faire un test pour mon administration</button>
+                            </div>
+                        </div>
+                        <div id="thank-you-message" class="notification success" style="display: none">Merci ! Nous
+                            revenons
+                            vers vous au plus vite.
+                        </div>
+                    </form>
+                </div>
+            </div>
+            <div class="column illustration">
+                <img src="img/file-d-attente.png" alt="File d'attente">
+            </div>
+        </div>
+    </div>
+</div>
+
+<section class="">
+    <div class="container">
+        <h2 class="section__title"><a href="#">37 administrations</a> ont déjà opté pour API Particulier avec des
+            résultats comme :</h2>
+        <div class="row">
+            <div class="card">
+                <div class="card__content">
+                    <p style="font-size: 1.3em; font-style: italic; text-align: center">
+                        "En 2014, nos usagers posaient une demi-journée pour effectuer leur démarche en mairie.
+                        Aujourd'hui, ils le font en ligne en 5 minutes."
+                    </p>
+                    <p style="text-align: right;">
+                        <i>- Simon Pierre, DGA action à la population, Clamart</i>
+                    </p>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card__content">
+                    <p style="font-size: 1.3em; font-style: italic; text-align: center">
+                        "Maitenant que le traitement est automatisé, j'ai enfin le temps d'aider les usagers en
+                        difficulté, dans certains cas je peux les contacter de manière proactive."
+                    </p>
+                    <p style="text-align: right;">
+                        <i>- Aurélie, directrices affaires générales, Clamart</i>
+                    </p>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card__content">
+                    <p style="font-size: 1.3em; text-align: center">
+                        <b>563 178</b> justificatifs papier évités pour le mois de septembre 2018
+                    </p>
+                </div>
+            </div>
+        </div>
+
+        <p class="section__subtitle"></p>
+    </div>
+</section>
+
+<section class="section-white">
+    <div class="container" style="font-size: x-large">
+        <p>
+            La lenteur d'une démarche administrative vient du traitement des pièces justificatives. Un usager doit régulièrement se rendre dans sa mairie pour fournir des informations qu'il a déjà transmis à d'autres administrations comme : son revenu fiscal de référence, son quotient familial CAF, la composition de sa famille, etc.
```

Comment:
Suggestions : 
La lenteur d'une démarche administrative vient souvent du temps imparti au traitement des pièces justificatives. Un usager doit régulièrement se rendre dans sa mairie pour fournir des informations qu'il a déjà transmis à d'autres administrations comme : 
 - son revenu fiscal de référence
 - son quotient familial CAF
 - la composition de sa famille
 - ...

Les pièces justificatives rendent les démarches plus complexes, augmentent les risques d'erreur et demandent un traitement manuel de chaque dossier.Faire un formulaire en ligne pour soumettre ces pièces justificatives ne résout pas le problème : un agent doit toujours passer du temps pour valider et saisir l'information fournie par l'usager. 

API Particulier permet d'entièrement automatiser les démarches en récupérant directement l'information à la source dans les bases de la DGFIP et de la CAF. L'information est certifiée et vous affranchit du besoin de demander et de valider des pièces justificatives. Votre démarche est alors 100% dématérialisée.

Vous pouvez utiliser API Particulier pour toutes vos démarches : transport, cantine, culture, sport, etc. À partir des informations fiscale, familiale et géographique de vos usagers, vous pouvez calculer automatiquement les montants de vos aides.

Nous voudrions limiter au maximum le recours aux pièces justificatives dans l'administration. Si vous avez la moindre question avant de commencer, n'hésitez pas à nous contacter.

Reply:
I have taken some of your suggestion into account. We should iterate on this.